scalableminds/webknossos
# Merger mode tasks

I did the backend changes myself (essentially copying what @philippotto did for the allowedMagnifications :)) and everything works as expected, but maybe @fm3 could have a quick look anyways.

During testing, I noticed that when deactivating the merger mode, the mapping is still active (need to activate/deactivate the ID mapping setting for it to go away). I would have expected it to be deactivated as well, but it's the same behavior on the current master, so maybe that's fine.

@philippotto Would be great if you could take this PR from here on and merge it if there are no major issues. Thanks a lot :heart: 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a task type with the `Merger Mode` checkbox checked and create a task with that task type
  - Get that task and trace, the merger mode should be automatically activated and it should not be possible to deactivate it. Merger mode should work as expected.
- Open a normal skeleton/hybrid tracing in a dataset with a segmentation layer.
  - Set some nodes or import an NML, then activate merger mode - should work as before.

### Issues:
- fixes #4090 

------
